### ISSUE TITLE
Suggestion: allow blockquotes as list item contents?

### ISSUE THREAD

**Issue by DivineDominion** [fletcher/MultiMarkdown-6#215]
![image](https://user-images.githubusercontent.com/59080/125169296-ca054e00-e1a9-11eb-81dd-de560a86d059.png)

CommonMark allows blockquotes at the start of lists. I think this might be useful in deeply nested outlines where an item is a citation.

This might be cumbersome to add if I understand the set-up of MMD6 correctly, since both list items and blockquotes are _block_ level items, and I believe one cannot nest these?

**Comment by fletcher** [comment]
Yeah....  This is highly dependent on what sort of parser you are using....

Markdown does not support this.

MultiMarkdown v5 did (it's parser was very recursive, so it was easy to "trash" the outer level and re-parse the inner level, but this did cause a performance hit)

MultiMarkdown v6 does not support this, as you note.

I am not philosophically opposed to it, though I'm not sure how necessary it is.  To me it's currently more of a pragmatic issue of how much time would it take to implement vs other things I am working on.  Additionally, how much would it impact MultiMarkdown v6 performance, and would it increase the risk of other parsing errors/issues.
